Resources/public/js/views/ez-universaldiscoveryview.js

```diff
@@ -559,7 +559,7 @@ YUI.add('ez-universaldiscoveryview', function (Y) {
                     return [
                         new Y.eZ.UniversalDiscoveryBrowseView({
                             bubbleTargets: this,
-                            priority: 100,
+                            priority: 1000,
```

Comment:
why do you change this priority ? Actually, you should completely remove UniversalDiscoveryBrowseView from the list (and adds a deprecation warning when the corresponding module is loaded)